app/assets/stylesheets/admin/pages/subscription_line_items.css.scss

```diff
@@ -0,0 +1,11 @@
+#add-line-item {
+  fieldset {
+    td.vertical_align_top {
```

Comment:
This is definitely an improvement and semantic naming would be even better. Or in this case, do you even need the class? Is the `#add-line-item fieldset td` selector enough?

Reply:
the third column with the button cannot have the class.
and I removed td in the following commit when reviewing rubocop issues.